packages/exemplar-sample/package.json

```diff
@@ -28,10 +28,12 @@
   "homepage": "https://github.com/godaddy/exemplar/tree/master/packages/exemplar-sample",
   "dependencies": {
     "@exemplar/storybook-react": "^1.1.0",
-    "@exemplar/storybook-react-native": "^1.0.1",
+    "@exemplar/storybook-react-native": "^1.0.0",
     "@storybook/addon-knobs": "^5.1.9",
     "@storybook/addons": "^5.1.9",
+    "@storybook/core": "^5.1.11",
     "@storybook/react": "^5.1.9",
+    "core-js": "^3.2.1",
```

Comment:
This was specifically needed? That seems a bit weird.

Reply:
It was. After fixing the `@exemplar/storybook-react-native` version to one that's actually available and installing the babel presets, all the builds for the web examples started failing due to the following exception:

```
ERROR in ./examples/web/yo-dawg.js
Module not found: Error: Can't resolve 'core-js/modules/es.symbol.iterator' in '/Users/rferreira/workspace/temp/exemplar/packages/exemplar-sample/examples/web'
 @ ./examples/web/yo-dawg.js 3:0-44
 @ ./examples/web sync ^\.\/([-\w]+)\.js$
 @ ./node_modules/@exemplar/storybook-react/load-examples.js
 @ multi ./node_modules/@storybook/react/node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/react/node_modules/@storybook/core/dist/server/preview/globals.js ./examples/.setup/config.js ./node_modules/@exemplar/storybook-react/load-examples.js (webpack)-hot-middleware/client.js?reload=true
```

I fixed it by installing core-js

Reply:
I can still repro. Can try other suggestions for fixing it.

Reply:
Weird. Seems fine though